plugins/sfIsadPlugin/modules/sfIsadPlugin/actions/stylesheetComponent.class.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class sfIsadPluginStylesheetComponent extends sfComponent
+{
+  public function execute($request)
+  {
+    // If institutional scoping is on, repo comes from the search-realm
+    if (sfConfig::get('app_enable_institutional_scoping') && sfContext::getInstance()->user->hasAttribute('search-realm') )
+    {
+      $repository = QubitRepository::getById(sfContext::getInstance()->user->getAttribute('search-realm'));
+    }
+    // If the feature is off, fall back to the repository component rules.
+    else if (!sfConfig::get('app_enable_institutional_scoping'))
+    {
+      if (!isset($request->getAttribute('sf_route')->resource))
+      {
+        return sfView::NONE;
+      }
+
+      $resource = $request->getAttribute('sf_route')->resource;
+
+      switch (true)
```

Comment:
heheh, I hadn't seen this in the wild before but I remember someone arguing for its case. It works I guess :)